Java/Assinment_5/First.java

```diff
@@ -0,0 +1,22 @@
+package com.company.data;
+
+public class First {
+
```

Comment:
As mentioned earlier please give proper namings for classes, variables and methods.

Reply:
Done !